fgrpc/grpcrunner.go

```diff
@@ -51,6 +58,14 @@ func Dial(o *GRPCRunnerOptions) (conn *grpc.ClientConn, err error) {
 	case strings.HasPrefix(o.Destination, fnet.PrefixHTTPS):
 		creds := credentials.NewTLS(&tls.Config{InsecureSkipVerify: o.Insecure}) // nolint: gosec // explicit flag
 		opts = append(opts, grpc.WithTransportCredentials(creds))
+	case strings.HasPrefix(o.Destination, fnet.PrefixXDS) && strings.EqualFold(os.Getenv(xdsSecuritySupportEnv), "true"):
```

Comment:
given previous version of fortio didn't know about xds:// anyway, why add the env check in addition to the prefix check?

Reply:
I was under the impression gRPC would reject this option if the env wasn't set but it actually seems to be fine.